.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": []
```

Comment:
- [ ] What is this vscode related settings ?

Reply:
Maybe while looking to fix prettier it poped up. I removed it